bibliography.bib

```diff
@@ -178,6 +178,35 @@ @InProceedings{clusterdata:Reiss2012
 
 
 
+################ 2023
+@INPROCEEDINGS{clusterdata:Tuns2023,
+  author = {Tuns, Adrian-Ioan and Spătaru, Adrian},
+  booktitle = {2023 25th International Symposium on Symbolic and Numeric Algorithms for Scientific Computing (SYNASC)},
+  title = {Cloud Service Failure Prediction on Google’s Borg Cluster Traces Using Traditional Machine Learning},
```

Comment:
Please protect the proper names with braces:

{Google Borg}

Reply:
I am not sure if I resolved this correctly, so please do check again. Thank you! 😄